website/content/docs/auth/login-mfa.mdx

```diff
@@ -0,0 +1,185 @@
+---
+layout: docs
+page_title: Multi-Factor Authentication (MFA) for Login - Auth Methods
+description: |-
+  Multi-factor authentication (MFA) is supported for several authentication
+  methods.
+---
+# Login MFA
+
+Vault has support for Multi-factor Authentication (MFA) for authenticating to
+an auth method, using different authentication types. We use the term `Login MFA` to distinguish
+this feature and the [Vault Enterprise MFA](/docs/enterprise/mfa).
+Login MFA is built on top of the Identity system of Vault.
+
+## MFA Types
+
+Login MFA in Vault can be of the following types.
+
+- `Time-based One-time Password (TOTP)` - If configured and enabled on a login path,
+  this would require a TOTP passcode along with a Vault token, to be presented
+  while invoking the API login request. The passcode will be validated against the
+  TOTP key present in the identity of the caller in Vault.
+
+- `Okta` - If Okta push is configured and enabled on a login path, then the enrolled
+  device of the user will get a push notification to approve or deny the access
+  to the API. The Okta username will be derived from the caller identity's
+  alias.
+
+- `Duo` - If Duo push is configured and enabled on a login path, then the enrolled
+  device of the user will get a push notification to approve or deny the access
+  to the API. The Duo username will be derived from the caller identity's
+  alias. Note that Duo could also be configured to use passcodes for authentication.
+
+- `PingID` - If PingID push is configured and enabled on a login path, then the
+  enrolled device of the user will get a push notification to approve or deny
+  the access to the API. The PingID username will be derived from the caller
+  identity's alias.
+
+## Login MFA Procedure
+
+Login MFA can be configured to further secure authenticating to an auth method. To enable login
```

Comment:
**Recommendation**: Try and avoid spitting infinitives: '...to secure' has been split by the infinitive 'further'. 
**Suggestion**: Login MFA can be configured to secure further authenticating to an auth method.